classes/item.rb

```diff
@@ -0,0 +1,39 @@
+require 'date'
+
+class Item
```

Comment:
- [ ] As was requested by the previous reviewer and it's a requirement for this Capstone. Kindly add a test for the methods missing on this class Item

***Nowadays, the market has a lot of developers that are good at writing code, but fewer developers that write tests. Remember that besides this being a requirement for the Capstone, it's a good practice and it's really good to show your projects with a good tests suite*** 💯 💯 

![image](https://github.com/unorjikingsley/Catalog-of-my-things/assets/93630700/6f017871-7df3-4138-bed7-4623bfb217c6)

![image](https://github.com/unorjikingsley/Catalog-of-my-things/assets/93630700/260f5c81-31c2-4ff1-9fd8-ac552c85e2a1)